build.rs

```diff
@@ -21,6 +22,10 @@ fn main() {
         out_path,
         "dynamodb",
     );
+    serde_generate(
```

Comment:
I actually have a local branch that refactors all of this, so no need to clean it up here too.  I'll put out a PR for that shortly.